apps/ui/src/views/User.vue

```diff
@@ -93,7 +93,14 @@ watch(
           proposal_count: 0,
           vote_count: 0,
           created: Date.now() / 1000,
-          name: (await getNames([userId]))?.[userId]
+          name: (await getNames([userId]))?.[userId] || '',
+          about: '',
+          avatar: '',
+          cover: '',
+          twitter: '',
+          github: '',
+          lens: '',
+          farcaster: ''
```

Comment:
I think it conflicts with this PR: https://github.com/snapshot-labs/sx-monorepo/pull/433

https://github.com/snapshot-labs/sx-monorepo/blob/06646af446b12ea31bafb73f0ecfb07eb1933947/apps/ui/src/networks/offchain/api/index.ts#L384-L386

Most likely we don't need placeholder now (but we should fill in missing fields in `loadUser`)

Reply:
Removed the placeholder, as offchain api will always return a `User` object.

Also added a check to prevent loading user when the address is not valid